rholang/examples/token.rho

```diff
@@ -29,7 +29,7 @@ new token in {
                                     balance_of_sender >= amount,
                                     amount > 0,
                                     balance_of_receiver + amount > balance_of_receiver] with
-                                    <true, true, true> => {
+                                    [true, true, true] => {
```

Comment:
This is already committed. Why am I seeing it again?

Reply:
Done